app/actions/remote/post.ts

```diff
@@ -552,3 +560,81 @@ export const togglePinPost = async (serverUrl: string, postId: string) => {
         return {error};
     }
 };
+
+export async function fetchPostThread(serverUrl: string, postId: string, fetchOnly = false) {
+    let client: Client;
+    try {
+        client = NetworkManager.getClient(serverUrl);
+    } catch (error) {
+        return {error};
+    }
+
+    try {
+        const data = await client.getPostThread(postId);
+        return processPostsFetched(serverUrl, ActionType.POSTS.RECEIVED_POST_THREAD, data, fetchOnly);
+    } catch (error) {
+        forceLogoutIfNecessary(serverUrl, error as ClientErrorProps);
+        return {error};
+    }
+}
+
+export async function fetchPostsAround(
+    serverUrl: string,
+    channelId: string,
+    postId: string,
+    perPage = 10,
```

Comment:
1. this should be a constant
2. is fine to keep it at 10 but look at my comment on the permalink view